pilot/pkg/bootstrap/server.go

```diff
@@ -278,6 +278,9 @@ func (s *Server) initMesh(args *PilotArgs) error {
 
 // initMixerSan configures the mixerSAN configuration item. The mesh must already have been configured.
 func (s *Server) initMixerSan(args *PilotArgs) error {
+	if s.mesh == nil {
```

Comment:
This shouldn't be necessary, but it's fine.  `initMesh` should guarantee that `s.mesh != nil`.